src/components/ArticleV3/Article.js

```diff
@@ -1,84 +1,64 @@
 /* eslint-disable no-unused-vars */
 import React from 'react'
-import { Container, Col, Row } from 'react-bootstrap'
+import { Container } from 'react-bootstrap'
 import { ArticleThebeProvider, useArticleThebe } from './ArticleThebeProvider'
-import { BootstrapColumLayout, BootstrapMainColumnLayout } from '../../constants'
 import SimpleArticleCell from './SimpleArticleCell'
 import { useNotebook } from './hooks'
+import { ExampleErrorTray } from './ExampleErrorTray'
 
 const Article = ({ mode = 'local', url = '', ipynb = { cells: [], metadata: {} }, ...props }) => {
   const { starting, error, ready, connectAndStart, restart } = useArticleThebe()
-  const { paragraphs, notebook, executing, executeAll, executeSome, clear } = useNotebook(
-    url,
-    ipynb,
-  )
+  const { paragraphs, notebook, executing, executeAll, clear } = useNotebook(url, ipynb)
   return (
     <Container>
-      <div style={{ paddingTop: 150 }}></div>
-      <Row>
-        <Col {...BootstrapMainColumnLayout}>
-          {!starting && !ready && (
+      <div style={{ paddingTop: 120 }}></div>
+      <div style={{ position: 'sticky', top: 100, zIndex: 10 }}>
+        {!starting && !ready && (
+          <button
+            style={{ margin: '4px', color: 'green' }}
+            disabled={starting || ready}
+            onClick={connectAndStart}
+          >
+            Start
+          </button>
+        )}
+        {starting && (
+          <span style={{ display: 'inline-block', marginLeft: '4px' }}>Starting...</span>
+        )}
+        {error && <ExampleErrorTray error={error} />}
```

Comment:
errors that occur in any cell will be returned at the top level here, and can be used to show a top level / global error icon, or message. Simple example included here